### ISSUE TITLE
"open app" does not function when TalkBack is enabled

### ISSUE THREAD

**Issue by dmochak** [ampproject/amphtml#4941]
## Short description of your issue:

The install banner does not redirect to the play store with the TalkBack screenreader active,.
## How do we reproduce the issue?

1. Activate TalkBack (settings > accessibility > TalkBack)
2. https://amp-mk-3.herokuapp.com/examples/viewer.html
3. Activate the AMP install banner by selecting "two"
4. Double tap "open in app."

expected result: the user should be taken to the play store entry for the app.

Actual result: nothing happens

## What browsers are affected?
Android (Chrome) 

## Which AMP version is affected?

Powered by AMP ⚡ HTML – Version 1473296795712
